reactor-core/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
 
   testCompile "junit:junit:$jUnitVersion"
   testCompile "org.junit.jupiter:junit-jupiter-api:${jUnitJupiterVersion}"
+  testCompile "org.junit.platform:junit-platform-launcher:1.6.1"
```

Comment:
Have you considered importing the [`junit-bom`](https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-junit-bom) to manage all JUnit 5 artifact versions?

Reply:
That's a good suggestion. I will check what we have for importing BOMs in reactor-core's build.

Reply:
OK. I'll let you investigate that separately, and I'll go ahead and mark my suggestions as approved.